### ISSUE TITLE
Release new version

### ISSUE THREAD

**Issue by waipeng** [puppetlabs/puppetlabs-kubernetes#679]
## Use Case
The current version [v8.0.0](https://github.com/puppetlabs/puppetlabs-kubernetes/commits/v8.0.0) is released about a year ago and there are 48 changes since. Some of the changes are really useful, like bumping dependency versions which will allow us to use newer libraries, etc.

## Describe the Solution You Would Like
Release a new version.

## Additional Context
I am happy to help with additional PRs to help with this.

**Comment by deric** [comment]
Yes, that would be great. There are [quite a few changes already](https://github.com/puppetlabs/puppetlabs-kubernetes/compare/v8.0.0...main) and few more PR for next major release. cc @Ramesh7 @GSPatton @david22swan

**Comment by waipeng** [comment]
v8.1.0 has been released in Dec, closing this issue
